### PR TITLE
Notion: fix of some elements that are not inverted when using the darkmode

### PR DESCRIPTION
**Pull request by august159**

### CODE REVIEW

**Comment by august159** [comment]
I am confident that the selectors are correct. I am less sure of which dynamic variable to use.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -11695,6 +11695,16 @@ CSS
...
     background-color: var(--darkreader-neutral-background) !important;
 }
+.notion-page-block
+{
+color: var(--darkreader-inline-text) !important;
+}
+div[contenteditable="true"],
+.notion-page-content div
+{
+color: var(--darkreader-inline-text) !important;
```
[the remark this answers is not in the document]
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:11705]: See: https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md#dynamic-variables

[the remark this answers is not in the document]
Comment (august159) [reply src/config/dynamic-theme-fixes.config]: I have replaced `--darkreader-inline-text` with `--darkreader-neutral-text`

**Review by august159 — COMMENTED** [review]
I have replaced `--darkreader-inline-text` with `--darkreader-neutral-text`

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -11695,6 +11695,16 @@ CSS
...
+.notion-page-block
+{
+color: var(--darkreader-neutral-text) !important;
+}
+div[contenteditable="true"],
+.notion-page-content div
+{
+color: var(--darkreader-neutral-text) !important;
+-webkit-text-fill-color: var(--darkreader-neutral-text) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11707]:
```suggestion
.notion-page-block,
div[contenteditable="true"],
.notion-page-content div {
    color: var(--darkreader-neutral-text) !important;
}
```

**Review by Gusted — APPROVED** [review]

**Pull request merged**